public/index.html

```diff
@@ -1,44 +1,61 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="description" content="Cryptocurrency exchange. Fiat-enabled, Open-source, API-based. Anonymous trade of BTC (Bitcoin), ETH (Ethereum), LTC (Litecoin) and latest altcoins. Buy, sell and mix your crypto-currency.">
-    <meta name="keywords" content="n.exchange,cryptocurrency exchange,cryptocurrencies,bitcoin exchange,ethereum exchange,fiat exchange,buy bitcoin,buy ethereum,open-source exchange,anonymous exchange,sell cryptos,buy cryptos,trade cryptos,exchange cryptos, buy btc, sell btc, buy eth, sell eth">
-    <meta name="robots" content="index, follow">
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+    <meta
+      name="description"
+      content="Cryptocurrency exchange. Fiat-enabled, Open-source, API-based. Anonymous trade of BTC (Bitcoin), ETH (Ethereum), LTC (Litecoin) and latest altcoins. Buy, sell and mix your crypto-currency."
+    />
+    <meta
+      name="keywords"
+      content="n.exchange,cryptocurrency exchange,cryptocurrencies,bitcoin exchange,ethereum exchange,fiat exchange,buy bitcoin,buy ethereum,open-source exchange,anonymous exchange,sell cryptos,buy cryptos,trade cryptos,exchange cryptos, buy btc, sell btc, buy eth, sell eth"
+    />
+    <meta name="robots" content="index, follow" />
     <meta http-equiv="content-type" content="text/html;charset=utf-8" />
-    <meta property="og:type" content="business.business">
-    <meta property="og:title" content="n.exchange - The planet's fastest and easiest Cryptocurrency exchange">
-    <meta property="og:description" content="Cryptocurrency exchange for your virtual currency needs. fiat-enabled, Open-source, API-based and anonymous conversion of crypto majors BTC (Bitcoin), ETH (Ethereum), LTC (Litecoin) and latest altcoins for trading as well as Fiat(USD, GBP, EUR, JPY). Buy, sell and mix your crypto-currency.">
-    <meta property="og:url" content="https://n.exchange/">
-    <meta property="og:image" content="https://n.exchange/img/expand.png">
+    <meta property="og:type" content="business.business" />
```

Comment:
how difficult it is to make individual properties here with react helmet? @punit2502